### ISSUE TITLE
2D detector

### ISSUE THREAD

**Issue by Willy0919** [xinzhuma/patchnet#15]
Thanks for your excellent work and sharing source code with us.

Since the performance is highly dependent on the 2D-detector, I am curious whether you can provide the 2D detection results on test set. I will be very appreciated if you can share the 2D detector on test set with us :)

**Comment by mrsempress** [comment]
Could you please provide the 2D detection results on test set or the 2D detection algorithm you use?

**Comment by xinzhuma** [comment]
We will not publish test set results directly. This may cause unfair comparison for others, someone may directly submit the results to the 2D detection benchmark. The results on the validation set are okay because they will not undermine the fairness of this research community and can help others to reproduce our results. If you want to use our model on the test set, you need to train the 2D detector by yourself. About the details of the 2D detector, you can refer to the supp of this CVPR paper: Frustum PointNets for 3D Object Detection from RGB-D Data, cvpr18.
